### ISSUE TITLE
Scrollable vertical axis on horizontal bar chart

### ISSUE THREAD

**Issue by prasadsawant12** [novus/nvd3#1909]
I am implementing nvd3-horizontal bar chart with 50 to 60 records .
I want to keep horizontal axis fixed and vertical axis scrollable.
But there is no option to do this. please help

**Comment by bgth** [comment]
please share a jsfiddle with an example of what you have.

**Comment by prasadsawant12** [comment]
Refer below plunker link.
http://plnkr.co/edit/ccBLTuOciLVbhxEWC04S?p=preview

**Comment by liquidpele** [comment]
For a scrollable axis, you'll have to manually set the hight of the chart using CSS, and put it within a div that scrolls.   nvd3 will not put scroll bars by default.

**Comment by prasadsawant12** [comment]
Yes  But the problem with this solution is, when we scroll vertical axis, horizontal axis also move.
I want to keep Horizontal axis fixed, so it will always be visible even if we scroll vertical axis .

For eg, 
![scrrenshot](https://cloud.githubusercontent.com/assets/24837942/22539832/a6c7885e-e942-11e6-96ac-54e89032d0d8.JPG)

**Comment by Docteur-RS** [comment]
Can't you set the "overflow-x" property  to "hidden" to prevent the x axis from moving ?
